adaptive_components/example/.firebase/hosting.YnVpbGQvd2Vi.cache

```diff
@@ -0,0 +1,12 @@
+favicon.png,1600088871060,fcc7c4545d5b62ad01682589e6fdc7ea03d0a3b42069963c815c344b632eb5cf
```

Comment:
Hmm this seems like the kind of file that should be gitignored

Reply:
Added